installer/roles/provision-mobile-catalog/defaults/main.yml

```diff
@@ -0,0 +1,2 @@
+---
+ansible_service_broker_namespace: openshift-ansible-service-broker
```

Comment:
is this something that the user may want to change

Reply:
I think so yes, I will keep this as default but allow to be overwritten by playbook argument